.vscode/settings.json

```diff
@@ -9,6 +9,7 @@
     ],
     "tailwindCSS.classAttributes": ["class", "className", "style", ".*Style"],
     "prettier.enable": false,
+    "editor.formatOnSave": true,
```

Comment:
This is intended?

Reply:
Yes, because otherwise it doesn't format on save in my VSCode and that's a pain 😓